launch.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
We should add a little documentation for this or at least a docstring at top of file to explain what this does!